test/e2e/apps/statefulset.go

```diff
@@ -125,7 +125,7 @@ var _ = SIGDescribe("StatefulSet", func() {
 
 		// This can't be Conformance yet because it depends on a default
 		// StorageClass and a dynamic provisioner.
-		ginkgo.It("should provide basic identity", func() {
+		ginkgo.It("should provide basic identity [pre-migration]", func() {
```

Comment:
I think the tag should be something along the lines of gce-pd. Maybe [Feature: gce-pd] which would then cause them to be automatically skipped in many circumstances?

@jpbetz 
@msau42 
You two probably know the right convention here.

Reply:
make sense to me.

Reply:
It's not a matter of making sense, it's a matter of being consistent with tagging conventions used elsewhere, eg the [Feature: XX] tags.

Also, "pre-migration" is by definition a temporary state. These tags will exist for years and years, and it will be annoying to have them refer to a situation that only existed for a matter of months. Really the thing we want to express is that (1) the tests are dependent on the GCE cloud provider, or (2) the tests are dependent on something like a default storage class that supports dynamic provisioning. Lumping those two situations under the same tag will be a maintenance burden, and these e2e tests are already messy enough.

Reply:
This test runs on other clouds too. I think I would use a generically named tag like [storage-provider-required] or something like that.

Reply:
Or maybe [Feature:CloudProvider]

Reply:
changed tag to `[Feature:CloudProvider]`